geomstats/hypersphere.py

```diff
@@ -135,6 +135,24 @@ def random_uniform(self, n_samples=1):
 
         return point
 
+    def random_uniform_unbounded(self, n_samples=1):
+        """
+        Sample in the whole of the Hypersphere with the uniform
+        distribution.
+        """
+        size = (n_samples, self.dimension)
+        spherical_coord = gs.random.rand(*size) * gs.pi
+        spherical_coord[:, -1] *= 2
```

Comment:
Matrix multiplication with gs.pi * gs.array([1, 2]) would make the code slightly more readable.

Reply:
Since it depends on the dimension, it would require to add ```* gs.hstack([gs.ones(self.dimension - 1), 2])``` which seems more involved. But if you still think it is more readable (I trust your experience !) I can change it.